src/main/java/frc5124/robot2020/RobotContainer.java

```diff
@@ -51,40 +52,30 @@
 
   private NetworkTableEntry shuffleboardButtonBooleanEntry;
 
-
-  /**
-   * The container for the robot.  Contains subsystems, OI devices, and commands.
-   */
   public RobotContainer() {
     configureSubsystems();
+    configureButtonBindings();
     configureShuffleboard();
     configureDefaultCommands();
-    configureButtonBindings();    
+    
   }
 
   private void configureSubsystems() {
     camera = new Camera();
     controlPanel = new ControlPanel();
     driveTrain = new DriveTrain();
     hanger = new Hanger();
-    intake = new Intake();
+    intake = new Intake(); 
     loader = new Loader();
     shooter = new Shooter();
     turret = new Turret();
-
   }
 
   private void configureButtonBindings(){
-      operatorRB.whileHeld(new IntakeBall(intake));
-      operatorLB.whileHeld(new OuttakeBall(intake));
-      operatorA.whileHeld(new IntakePivotDown(intake));
-      operatorY.whileHeld(new IntakePivotUp(intake));
-      operatorUp.whileHeld(new LiftUp(hanger));
-      operatorDown.whileHeld(new LiftDown(hanger));
   }
 
   private void configureDefaultCommands(){
-    //driveTrain.setDefaultCommand(new JoystickTankDrive(OI.driver, driveTrain));
+    turret.setDefaultCommand(new TargetTracker(turret));
```

Comment:
should not merge code into master that has a default command like this for the turret. it needs to be controlled by the drive team to determine when to target, not on all the time. Bind this to a button rather than default